src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,55 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
 public class SalaryInfo {
-    public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+    public String getSalaryInfo(
+            String[] names, String[] data,
+            String dateFrom, String dateTo) {
+
+        StringBuilder reportForPeriod = new StringBuilder("Report for period ");
+        reportForPeriod.append(dateFrom).append(" - ").append(dateTo);
+        SalaryInfo dateParser = new SalaryInfo();
```

Comment:
you don't need to create a separate instance of this class, remove this line

Reply:
fixed